piker/brokers/ib.py

```diff
@@ -1480,9 +1483,8 @@ async def trades_dialogue(
         for client in _client_cache.values():
             for pos in client.positions():
                 msg = pack_position(pos)
-                all_positions.setdefault(
-                    msg.symbol, []
-                ).append(msg.dict())
+                msg.account = accounts_def.inverse[msg.account]
```

Comment:
Added in this: sends the user defined (in `brokers.toml`) account *name* now instead of the actual account number/value over the wire.

This probably makes more sense anyway once a more distributed setup is going on (`brokerd` on diff hosts then GUI apps).